### PR TITLE
make validations class methods in the class they validate

### PR DESCRIPTION
**Pull request by elrayle**

### CODE REVIEW

File: lib/hydra/pcdm/models/concerns/collection_behavior.rb [hunk 1]
```diff
@@ -7,6 +7,11 @@ module CollectionBehavior
     included do
       type RDFVocabularies::PCDMTerms.Collection  # TODO switch to using generated vocabulary when ready
       aggregates :members, :predicate => RDFVocabularies::PCDMTerms.hasMember, :class_name => "ActiveFedora::Base"
+
+      def self.collection? collection
```
Comment (jcoyne) [inline lib/hydra/pcdm/models/concerns/collection_behavior.rb]: You want this to be in a ClassMethods module (within CollectionBehavior)

Comment (jcoyne) [inline lib/hydra/pcdm/models/concerns/collection_behavior.rb]: I wonder if this woudn't make more sense in the Hydra::PCDM module. e.g. `Hydra::PCDM.collection?(obj)`

File: lib/hydra/pcdm/models/concerns/object_behavior.rb [hunk 2]
```diff
@@ -8,9 +8,14 @@ module ObjectBehavior
       type RDFVocabularies::PCDMTerms.Object  # TODO switch to using generated vocabulary when ready
       aggregates :members, :predicate => RDFVocabularies::PCDMTerms.hasMember, :class_name => "ActiveFedora::Base"
 
+      def self.object? object
```
Comment (jcoyne) [inline lib/hydra/pcdm/models/concerns/object_behavior.rb]: Can we move this to the Hydra::PCDM module? e.g. `Hydra::PCDM.object?(obj)`

**Comment by elrayle** [comment]
@jcoyne I'm not sure I am following your suggestion.  Currently, there is not a class Hydra::PCDM.  There is a module at lib/hydra/pcdm.rb.

**Pull request closed without merging**